app.test.js

```diff
@@ -57,4 +57,28 @@ describe('get /api',()=>{
     })
 })
 })
-    
\ No newline at end of file
+    
+describe('get /api/articles',()=>{
+    test('should return 200 and an array of article objects with the correct properties',()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body})=>{
+
+            expect(typeof body).toBe('object')
+            const { articles } = body;
+            expect(articles.length).toBe(13)
+            articles.forEach((article)=>{
+            expect(typeof article).toBe('object')
+            expect(article).toHaveProperty('author')
+            expect(article).toHaveProperty('title')
+            expect(article).toHaveProperty('article_id')
+            expect(article).toHaveProperty('topic')
+            expect(article).toHaveProperty('created_at')
+            expect(article).toHaveProperty('votes')
+            expect(article).toHaveProperty('article_img_url')
+            expect(article).toHaveProperty('comment_count')
+            })
+        })
+    })
```

Comment:
Please also add a test to check the articles array is coming back sorted by date in descending order